apps/condo/domains/ticket/schema/ExportTicketsService.js

```diff
@@ -1,40 +1,33 @@
 const { GQLCustomSchema } = require('@core/keystone/schema')
 const access = require('@condo/domains/ticket/access/ExportTicketsService')
-const { Ticket } = require('@condo/domains/ticket/utils/serverSchema')
+const { Ticket, TicketComment, TicketChange, TicketStatus } = require('@condo/domains/ticket/utils/serverSchema')
 const moment = require('moment')
 const { createExportFile } = require('@condo/domains/common/utils/createExportFile')
-const get = require('lodash/get')
-const { checkOrganizationPermission } = require('@condo/domains/organization/utils/accessSchema')
-const { EMPTY_DATA_EXPORT_ERROR } = require('@condo/domains/ticket/constants/errors')
+const { has, get } = require('lodash')
 
+const { EMPTY_DATA_EXPORT_ERROR } = require('@condo/domains/ticket/constants/errors')
 const CHUNK_SIZE = 20
 const DATE_FORMAT = 'DD.MM.YYYY HH:mm'
 
+// TODO(zuch): if we add timeZone and locale to organization settings use organization timeZone instead of client's timezone
 const ExportTicketsService = new GQLCustomSchema('ExportTicketsService', {
     types: [
         {
             access: true,
-            type: 'input TicketExportExcelInput { where: TicketWhereInput!, sortBy: [SortTicketsBy!] }',
+            type: 'input ExportTicketsToExcelInput { where: TicketWhereInput!, sortBy: [SortTicketsBy!], timeZone: String! }',
         },
         {
             access: true,
-            type: 'type TicketExportExcelOutput { status: String!, linkToFile: String! }',
+            type: 'type ExportTicketsToExcelOutput { status: String!, linkToFile: String! }',
         },
     ],
     queries: [
         {
             access: access.canExportTicketsToExcel,
-            schema: 'exportTicketsToExcel(data: TicketExportExcelInput!): TicketExportExcelOutput',
+            schema: 'exportTicketsToExcel(data: ExportTicketsToExcelInput!): ExportTicketsToExcelOutput',
             resolver: async (parent, args, context, info, extra = {}) => {
-                const { where, sortBy } = args.data
-                const organizationId = get(where, 'organization.id')
-                if (!organizationId) {
-                    throw new Error('[error] no organization id is passed')
-                }
-                const hasAccess = await checkOrganizationPermission(context.authedItem.id, organizationId, 'canManageTickets')
-                if (!hasAccess) {
-                    throw new Error('[error] you do not have access to this organization')
-                }
+                const { where, sortBy, timeZone } = args.data
```

Comment:
is it possible to check timeZone argument?

Reply:
Added DEFAULT_ORGANIZATION_TIMEZONE to organization common constants
Added "normalizeTimeZone" to timezone.js to common utils with spec. Works the same as "normalizePhone"
if passed timeZone is not valid will be used DEFAULT_ORGANIZATION_TIMEZONE